externs/ima.js

```diff
@@ -259,6 +259,9 @@ google.ima.Ad = class {
   /** @return {number} */
   getVastMediaWidth() {}
 
+  /** @return {string} */
+  getVastAdId() {}
+
```

Comment:
`google.ima.Ad` does not have such a method according to docs: https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/interface/google.ima.Ad

Reply:
@tykus160 thank you, I removed the method from the IMA and IMA Dai interfaces.

Keeping the functions to comply with `shaka.extern.IAd` but just returning empty string